src/runner/CMakeLists.txt

```diff
@@ -16,6 +16,9 @@ add_library(runner STATIC ${runner_sources} ${runner_headers})
 
 target_link_libraries(runner PUBLIC fileInput)
 
+# TODO might not need this
+target_link_libraries(runner PUBLIC utilities)
```

Comment:
pretty sure fileInput needs the utilities library so the dependencies better propagate through that or the other libraries.

Reply:
Both! fileInput didn't have utilities, and runner uses a couple things from utilities.